### ISSUE TITLE
HTTP Digest doesn't play well with others

### ISSUE THREAD

**Issue by brong** [cyrusimap/cyrus-imapd#1387]
From: [Ken Murchison](murch@andrew.cmu.edu)
Bugzilla-Id: 3515
Version: 2.5.x (next)
Owner: [Ken Murchison](murch@andrew.cmu.edu)

**Comment by dilyanpalauzov** [comment]
Is this still relevant?

**Comment by brong** [comment]
I don't think this matters - either clients have improved or just don't enable that.
